app/controllers/user/delete.js

```diff
@@ -0,0 +1,24 @@
+const { User } = require('../../models');
+
+module.exports = (req, res, next) => {
+    User.remove({ _id: req.params.id})
+        .exec()
+        .then((result) => {
+            console.log(result);
+            if (result.n) {
```

Comment:
what is result.n

Reply:
`result.n` is number of successfully removed documents.